### PR TITLE
Added Information about Thanos Contributor Hours

### PR DESCRIPTION
**Pull request by DARK-art108**
## Changes

Added info about Thanos Contributor Office Hours to Thanos Website. Issue #3534 
  - [x] Added Thanos-Office-Contributing-Hour.md file with all the Details.
  - [x] Added a Thanos-Office-Contributing-Hour section in Community.md

### CODE REVIEW

**Comment by DARK-art108** [comment]
@soniasingla Please review the PR

**Comment by bwplotka** [comment]
Please run `make docs` and commit changes fix CI issue (:

**Comment by soniasingla** [comment]
@DARK-art108 thanks for the PR :)

CI is failing, please visit [here](https://github.com/thanos-io/thanos/pull/3544/checks?check_run_id=1512375335), it will tell you what to do to fix the CI !!

And you can preview the website [here](https://deploy-preview-3544--thanos-io.netlify.app/tip/contributing/community.md/#thanos-contributor-office-hours), check out the [links](https://thanos.io/tip/contributing/Thanos-Contributor-Office-Hours.md/) you mentioned, it's not working. If anything is not clear, make sure to ask.

**Comment by DARK-art108** [comment]
@soniasingla @bwplotka  I am not getting how to fix DCO? I have also done the steps mentioned in the DCO details but it is throwing a rejected error please help me with it.

**Comment by soniasingla** [comment]
@DARK-art108 on first, checkout the branch `Thanos-Contri-Hours` on your terminal.

After that, perform these steps as mentioned in DCO details:
>git rebase HEAD~8 --signoff
>git push --force-with-lease origin Thanos-Contri-Hours

**Comment by DARK-art108** [comment]
@soniasingla the first rebasing cmd's executed successfully but in executing the push cmd it is throwing error 'failed to push some refs'.

**Comment by soniasingla** [comment]
@DARK-art108 this error is occurring because your branch is not rebased, can you please try rebasing before pushing?

**Comment by DARK-art108** [comment]
@soniasingla @bwplotka I have fixed everything and also the links are given in the community.md file will start working when the PR is merged as I have added the new file which is Thanos-Contributor-Office-Hours.md file which is currently not present.

File: docs/contributing/community.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ Everyone is welcome to join & add items to the agenda. The meeting happens in Zo
 
 If you have any suggestions, how we can run the meeting better, please let `@povilasv` or `@metalmatze` know in the CNCF slack.
 
+### Thanos Contributor Office Hours
+
+Thanos Contributor Office Hours are public weekly meetings focused on Thanos development and contributions. It’s meant for developers and maintainers to meet and get unblocked, pair review, and discuss development aspects of the Thanos and related projects. Read more [here](https://github.com/thanos-io/thanos/tree/master/docs/contributing/Thanos-Contributor-Office-Hours.md)
```
Comment (bwplotka) [inline docs/contributing/community.md:32]:
```suggestion
Thanos Contributor Office Hours are public weekly meetings focused on Thanos development and contributions. It’s meant for developers and maintainers to meet and get unblocked, pair review, and discuss development aspects of the Thanos and related projects. Read more [here](https://docs.google.com/document/d/137XnxfOT2p1NcNUq6NWZjwmtlSdA6Wyti86Pd6cyQhs/edit#heading=h.oy8b2jtupl8m)
```
Can we link Google doc instead?

File: docs/proposals/receive_split.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Thanos Remote Write
```
Comment (bwplotka) [inline docs/proposals/receive_split.md:2]: Is this relevant change?

**Review by bwplotka — CHANGES_REQUESTED** [review]
Also I think rebase did not work, this PR has unrelated changes.

File: docs/proposals/receive_split.md (hunk 2 above)
Comment (DARK-art108) [reply docs/proposals/receive_split.md:2]: This is not my change this is attached to my PR due to the wrong rebase, I have fixed it in the new PR #3552 .

File: docs/contributing/community.md (hunk 1 above)
Comment (DARK-art108) [reply docs/contributing/community.md:32]: Yes, we can I have done the requested changes in PR #3552

**Comment by soniasingla** [comment]
@DARK-art108 can you please close this PR?

**Pull request closed without merging**